### PR TITLE
[tests] fix Pixart Sigma tests

### PR DESCRIPTION
**Pull request by sayakpaul**
# What does this PR do?

Fixes PixArt Sigma tests. 

Cc: @lawrence-cj for awareness.

### CODE REVIEW

File: tests/pipelines/pixart_sigma/test_pixart.py [hunk 1]
```diff
@@ -336,23 +336,28 @@ def test_pixart_1024(self):
         image = pipe(prompt, generator=generator, num_inference_steps=2, output_type="np").images
 
         image_slice = image[0, -3:, -3:, -1]
-        expected_slice = np.array([0.0742, 0.0835, 0.2114, 0.0295, 0.0784, 0.2361, 0.1738, 0.2251, 0.3589])
+        expected_slice = np.array([0.4517, 0.4446, 0.4375, 0.449, 0.4399, 0.4365, 0.4583, 0.4629, 0.4473])
```
Comment (sayakpaul) [inline tests/pipelines/pixart_sigma/test_pixart.py:339]: Pretty sure these obtained on a different hardware.

Comment (lawrence-cj) [reply tests/pipelines/pixart_sigma/test_pixart.py:339]: I got the expected result on V100 GPU. But, I remember I passed the text before.

Comment (sayakpaul) [reply tests/pipelines/pixart_sigma/test_pixart.py:339]: Our runner is based on t4. But not a big problem.

Comment (lawrence-cj) [reply tests/pipelines/pixart_sigma/test_pixart.py:339]: Got it.

**Review by yiyixuxu — APPROVED** [review]
thanks!

**Pull request merged**